client/clientservice/src/main.cpp

```diff
@@ -31,6 +32,8 @@ int main(int argc, char** argv) {
   // TODO: Setup ConcordClientConfig and read from config file
   ConcordClientConfig config{};
   auto concord_client = std::make_unique<ConcordClient>(config);
+  auto metrics = std::make_shared<concordMetrics::Aggregator>();
+  concord_client->setMetricsAggregator(metrics);
```

Comment:
Nit: I assume there is no scenario when we'd like to reset the aggregator in the runtime.
If my assumption is correct, we can pass the metrics aggregator right to the client's ctor without adding a new method.
WDYT?

Reply:
The idea is that metrics are optional. If we have an aggregator than we use it. Otherwise, metrics are ignored.